src/clj/rems/api/blacklist.clj

```diff
@@ -0,0 +1,50 @@
+(ns rems.api.blacklist
+  (:require [clj-time.core :as time]
+            [compojure.api.sweet :refer :all]
+            [rems.api.schema :as schema]
+            [rems.db.blacklist :as blacklist]
+            [rems.util :refer [getx-user-id]]
+            [ring.util.http-response :refer [ok]]
+            [schema.core :as s]))
+
+(s/defschema BlacklistCommand
+  ;; TODO should these be namespaced keywords?
```

Comment:
The other command APIs don't use namespaced keywords, so it's okay the way it's now. We can add them later everywhere if needed.